scripts/install-deps.sh

```diff
@@ -27,7 +27,7 @@ scripts_dir="$(dirname "${BASH_SOURCE[0]}")"
 
 sudo apt-get -y install alsa-utils python3-all-dev python3-pip python3-numpy \
   python3-scipy python3-virtualenv rsync sox libttspico-utils ntpdate
-sudo pip3 install --upgrade pip virtualenv
+sudo pip3 install --upgrade pip virtualenv Rpi.GPIO
```

Comment:
@jordandrako I tackled the deps in another PR https://github.com/google/aiyprojects-raspbian/pull/35/files, you can remove the changes from this file :)